### ISSUE TITLE
fix: tab/shift-tab does not work as expected in MenuButton

### ISSUE THREAD

**Issue by adamint** [microsoft/fluentui-blazor#4090]
# 🐛 Bug Report

The MenuButton component is not accessible to keyboard users. When opening the MenuButton, entering tab should advance to the first element in the menu. Shift-tab should place focus back on the button.

## 💻 Repro or Code Sample

Can be reproduced in any of the MenuButton examples -> https://www.fluentui-blazor.net/MenuButton

## 💁 Possible Solution

Register an onkeydown listener on both the first menu item (if one exists) and the menu button. If focus is on the menu button, tab will place focus on the first item in the menu. If focus is on the first menu item, a shift-tab will place focus back on the button.

## 🔦 Context

Caught in manual testing. I will provide a fix

## 🌍 Your Environment

* OS & Device: [e.g. MacOS, iOS, Windows, Linux] on [iPhone 7, PC]
* Browser [e.g. Microsoft Edge, Google Chrome, Apple Safari, Mozilla FireFox]
* .NET and Fluent UI Blazor library Version [e.g. 8.0.2 and 4.4.1]
